audio-buffer-source-node/playbackrate/script.js

```diff
@@ -0,0 +1,49 @@
+const audioCtx = new AudioContext();
```

Comment:
For this example, I broke down in small suggestions the changes I did for working around the _Autoplay policy in Chrome_.
```suggestion
let audioCtx;
```